docs/coding-guidelines/csharp-coding-style.md

```diff
@@ -3,6 +3,30 @@ C# Coding Style
 
 We use the same [coding style](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md) and [EditorConfig](https://editorconfig.org "EditorConfig homepage") file (`.editorconfig`) used by [dotnet/corefx](https://github.com/dotnet/corefx) with the following differences:
 
+* **`var` must be used when `new`, `as`, or cast operator is used (and it can be used only with these operators).**
```

Comment:
Can we change the order of `as` and cast either here or in the following examples to be consistent?